### PR TITLE
docs(examples): update nuxt-langchain example

### PR DESCRIPTION
**Pull request by danielroe**
realised we forgot to update this example in https://github.com/vercel/ai/pull/295

### CODE REVIEW

File: examples/nuxt-langchain/.env.example [hunk 1]
```diff
@@ -1 +1 @@
-OPENAI_API_KEY=
+NUXT_OPENAI_API_KEY=
```
Comment (MaxLeiter) [inline examples/nuxt-langchain/.env.example:1]: is this being used?

**Review by MaxLeiter — CHANGES_REQUESTED** [review]

File: examples/nuxt-langchain/.env.example (hunk 1 above)
Comment (danielroe) [reply examples/nuxt-langchain/.env.example:1]: Yes, this is consumed by Nuxt runtime config.

Comment (MaxLeiter) [reply examples/nuxt-langchain/.env.example:1]: ah very cool, I didn't realize that's how useRuntimeConfig worked.

**Review by MaxLeiter — APPROVED** [review]
Thank you and good catch!

**Pull request merged**